learn/advanced/asynchronous_updates.md

```diff
@@ -108,12 +108,19 @@ Had the update failed, the response would have included an error message:
 ```json
 {
   "status": "failed",
-  "updateId": 1,
-  "type": { "name": "DocumentsAddition" },
-  "enqueuedAt": "2019-12-07T21:10:07.607581330Z",
-  "duration": 0.000048524,
-  "processedAt": "2019-12-07T21:10:20.511525620Z",
-  "error": "document id is missing"
+  "updateId": 0,
+  "type": {
```

Comment:
This should be in one line 👽

```suggestion
"type": { "name": "DocumentsAddition" },
```

Reply:
Is this right? Doesn't it break @bidoubiwa 's `one key/value per line` [principle](https://github.com/meilisearch/documentation/issues/872#issuecomment-803429279)?